presto-docs/src/main/sphinx/release/release-0.261.rst

```diff
@@ -0,0 +1,50 @@
+=============
+Release 0.261
+=============
+
+**Highlights**
+==============
+* Improve performance of sum and average aggregations on decimals.
```

Comment:
`Improve performance of :func:`SUM` and :func:`AVG` aggregate functions when used with ``DECIMAL`` type.

Reply:
Those function links don't work. I can basically never get them to work :-(